bucket/cataclysm-dda-tiles-experimental.json

```
@@ -1,35 +1,44 @@
 {
     "homepage": "https://cataclysmdda.org",
-    "description": "Turn-based survival game set in a post-apocalyptic world (with graphical tiles, experimental build)",
-    "version": "10442",
-    "license": "CC-BY-SA-3.0",
+    "description": "Roguelike in a post-apocalyptic world (with sprite-based graphics, experimental build)",
+    "version": "2021-10-30-0656",
+    "license": {
+        "identifier": "CC-BY-SA-3.0,GPL-2.0+,OFL-1.0,BSL-1.0,Zlib,MIT,BSD(libbacktrace)",
+        "url": "https://github.com/CleverRaven/Cataclysm-DDA/blob/master/LICENSE.txt"
+    },
```

For simplicity's sake, list only the main/most prominent/most restrictive license. In this case, it's CC-BY-SA-3.0:

```suggestion
    "license": "CC-BY-SA-3.0",
```